examples/shouldi/tests/test_dependency_check.py

```diff
@@ -0,0 +1,35 @@
+import pathlib
+
+from dffml.util.os import prepend_to_path
+from dffml.util.net import cached_download_unpack_archive
+from dffml.util.asynctestcase import AsyncTestCase
+
+from shouldi.dependency_check import run_dependency_check
+
+
+class TestRunDependencyCheckOp(AsyncTestCase):
+    @cached_download_unpack_archive(
+        "https://download.oracle.com/otn-pub/java/jdk/14+36/076bab302c7b4508975440c56f6cc26a/jdk-14_osx-x64_bin.tar.gz",
```

Comment:
Looks like the current link you have is java, but it's not the open source version, so when requesting the URL it gives us an error since we haven't accepted their license agreement. Instead, we'll use the open source one.

```suggestion
        "https://download.java.net/openjdk/jdk14/ri/openjdk-14+36_linux-x64_bin.tar.gz",
```